src/trace.h

```diff
@@ -0,0 +1,27 @@
+
+
+#ifndef BITCOIN_TRACE_H
```

Comment:
nit: 
- copyright header
- redundant empty lines here
- also i think we want this file under `src/util`